### PR TITLE
Add debian installation,Fixing ubuntu preseed file

### PR DESCRIPTION
**Pull request by Ara4Sh**

### CODE REVIEW

File: data/profiles/install-debian.ipxe [hunk 1]
```diff
@@ -0,0 +1,19 @@
+echo Starting Debian x64 installer for ${hostidentifier}
```
Comment (pengz1) [inline data/profiles/install-debian.ipxe]: If install-debian.ipxe is exactly the same with install-ubuntu.ipxe, you should modify install-ubuntu.ipxe to accommodate both, just like we have only one ipxe for CentOS/RHEL.

**Comment by lanchongyizu** [comment]
@Ara4Sh 
1) The ubuntu-preseed  file works well in our CI environment. Could you help to describe in details what issues exist in ubuntu-preseed? 
2) What's the differences between ubuntu-preseed and debian-preseed ? Is it possible that we can use the ubuntu-graph/ubuntu-preseed to install debian OS?

**Comment by RackHDCI** [comment]
The Copyright Check has failed!
The files listed below do not have a valid copyright statement:
- data/profiles/install-debian.ipxe - data/templates/install-debian/debian-interfaces - data/templates/install-debian/debian-preseed - data/templates/install-debian/debian-sources - data/templates/install-debian/debian.rackhdcallback - data/templates/install-debian/post-install-debian.sh - data/templates/install-ubuntu/ubuntu-preseed

**Comment by Ara4Sh** [comment]
I close this PR to add changes and create only one Task.Os.Install.Debian to install both ubuntu and debian.

**Pull request closed without merging**